docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
+
+Sets the nodes next boot device persistently
+
+device <dev> can be set to:
+
+  none  : Do not change boot device order
```

Comment:
Ipmitool overrides the boot order in BIOS. For example, if I set boot device to `bios`, the node will always boot into the BIOS regardless of the boot order, or if I set it to `disk` it will always boot from disk even if you manually set the boot order in BIOS. 

Setting boot device to `none` lets the node follow the boot order in BIOS.